show/main.py

```diff
@@ -187,6 +186,38 @@ def summary(interfacename):
         command = cmd_ifconfig
         run_command(command)
 
+
+@interfaces.group(cls=AliasedGroup, default_if_no_args=True)
+def transceiver():
+    pass
+
+interfaces.add_command(transceiver)
```

Comment:
Unnecessary - already added to `interfaces` via the decorator on line 190.

Reply:
done